lib/storage/src/content_manager/alias_mapping.rs

```diff
@@ -0,0 +1,100 @@
+use crate::content_manager::errors::StorageError;
+use segment::common::file_operations::{atomic_save_json, read_json};
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::fs;
+use std::io::Write;
+use std::path::{Path, PathBuf};
+
+pub const ALIAS_MAPPING_CONFIG_FILE: &str = "data.json";
+
+#[derive(Debug, Deserialize, Serialize, Clone, PartialEq)]
+struct AliasMapping(HashMap<String, String>);
```

Comment:
```suggestion
type Alias = String;

#[derive(Debug, Deserialize, Serialize, Clone, PartialEq)]
struct AliasMapping(HashMap<Alias, collection::CollectionId>);
```

Reply:
Here the alias attached to the collection name.
Is the `collection::CollectionId` representing the name of the collection in our codebase?

Reply:
Yes, it was intended to, but probably I forgot to update some types in Table Of Content to it.

Reply:
Alright, updated :+1: